lib/ui/user_profile_list/widgets/chat_list_item_tile.dart

```diff
@@ -97,10 +101,21 @@ class ChatListItemTile extends ConsumerWidget {
             ],
           ),
           child: InkWell(
-            onTap: () {
+            onTap: () async {
               if (onTap != null) {
                 onTap!();
               }
+              if (item.lastMessage != null) {
```

Comment:
[sug] This method definition looks a bit complex to be inside the template. I think it would help for readability to be defined as a named method in the class.